requirements.txt

```diff
@@ -1 +1,3 @@
 -e .
+colander
+pytz
```

Comment:
Why do you need these if they are in `setup.py` already?

Reply:
yup. That must go!